src/controllers/groupe.controller.ts

```diff
@@ -0,0 +1,83 @@
+import { NextFunction, Request, Response } from 'express';
+import { Group } from '@prisma/client';
+import { CreateGroupDto } from '@dtos/group.dto';
+import groupService from '@services/groups.service';
+
+class GroupController {
+  public groupService = new groupService();
```

Comment:
Use dependency injection for the services, into the constructor of this class. This makes it easier to replace dependencies with mocks during testing and ensures that dependencies are managed at a higher level, promoting modularity.